ompi/mca/osc/base/base.h

```diff
@@ -46,6 +46,28 @@ int ompi_osc_base_select(ompi_win_t *win,
                          int flavor,
                          int *model);
 
+
+int ompi_osc_base_iselect(ompi_win_t *win,
```

Comment:
I'm not sure this is needed - the "select" is choosing which module to use for supporting the RMA functionality. It is already a local (nonblocking) action, I think. So I'm pretty sure we don't have to split it into a begin piece and a complete piece.